tests/Feature/Api/RecitersApiTest.php

```diff
@@ -38,7 +39,7 @@ public function testGetReciter()
     public function testGetReciterBySlug()
     {
         /** @var Reciter $reciter */
-        $reciter = factory(Reciter::class)->create();
+        $reciter = factory(\App\Reciter::class)->create();
```

Comment:
Don't need this `\App\` here, because the class is imported at the top of the file.

Reply:
i will be removing that as i was testing a few things due to some errors i was receiving